jmix-templates/content/flowui/detail-dto/descriptor.xml

```diff
@@ -0,0 +1,34 @@
+<%
+dcId = "${entity.uncapitalizedClassName}Dc"
+dlId = "${entity.uncapitalizedClassName}Dl"
+
+def formXml = api.processSnippet('dto_form.xml',
+        ['entity': entity,
+        'api': api,
+        'dcId': dcId,
+        'formId': 'form'])
+%><?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<view xmlns="http://jmix.io/schema/flowui/view"
+      title="${messageKeys['detailTitle']}"
+      focusComponent="form">
+    <data>
+        <instance id="${dcId}"
+                  class="${entity.fqn}">
+            <loader id="${dlId}"/>
+        </instance>
+    </data>
+    <facets>
```

Comment:
At the time it handle loading, there won't be any entity in the container, test that it doesn't produce any exceptions